sdk-base/src/main/java/com/mapbox/maps/plugin/viewport/data/FollowPuckViewportStateOptions.kt

```diff
@@ -60,6 +60,7 @@ class FollowPuckViewportStateOptions private constructor(
    *
    * Defaults to [DEFAULT_STATE_ANIMATION_DURATION_MS] milliseconds
    */
+  @Deprecated("AnimationDurationMs is not needed any more, the transition will be handled properly internally.")
```

Comment:
I assume internally it's not held for now - that's why we can't drop it right now, correct?

Reply:
yes, the actual fix will land in rc.1, we don't want to rush the fix itself, Deprecate affects public API, so that's why we add the `Deprecated` in beta.1